src/core/_buttons.css

```diff
@@ -19,6 +19,11 @@ input[type="button"] {
   border-radius: var(--border-radius);
   border: var(--button-border);
   cursor: pointer;
+  -webkit-transition: border 0.2s;
```

Comment:
All prefixing is handled by PostCSS with Autoprefixer. Could you remove these from the src files, please, as they will be in the dist files. If some necessary prefixes are missing, you can report a bug at [Autoprefixer's issue tracker](https://github.com/postcss/autoprefixer/issues).